postgres/Cargo.toml

```diff
@@ -14,7 +14,8 @@ readme = "README.md"
 all-features = true
 
 [features]
-default = ["rt_tokio_1"]
+default = ["runtime", "rt_tokio_1"]
+runtime = ["tokio-postgres/runtime"]
```

Comment:
How about making this a dependency of the `rt_tokio_1` and `rt_async-std_1` feature instead? That way the default feature set will just stay the same and it won't break for users of `rs_async-std_1` either.

Reply:
I like this idea! Will update the PR.

Reply:
Or actually, on second thought, this won't work unfortunately. Because for example on WASM, we do want `rt_tokio_1` if we're running on Tokio, just not the `runtime` feature.